pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -636,8 +636,11 @@ class Build {
 
         context.node('built-in || master') {
             context.stage("installer") {
+                // Ensure master context workspace is clean
+                context.sh "rm workspace/target/* || true"
```

Comment:
Should that be rm -rf

Reply:
interesting, I actually copied that from further up... trying to think why it's done this way..?

Reply:
intention is just to clear any previous archives, so -r not required, always preferable to not to use -r
-f not really necessary as there is not terminal on a Jenkins job, but I think I will add it to be correct